src/main/java/com/GeorgesServer/app/App.java

```diff
@@ -4,10 +4,9 @@
 
 public class App {
     public static void main(String args[]) throws IOException {
-        String publicFolderPath = args[4];
+        String publicFolderPath = args[3];
         int port = Integer.parseInt(args[1]);
 
-        // The following should go into a config
```

Comment:
Although this line has been deleted, I still plan on creating a config and passing it into start(). This will clean up main, and allow me to configure my server in various ways.